src/Rules/UnusedFunctionParametersCheck.php

```diff
@@ -47,6 +47,14 @@ private function getUsedVariables($node): array
 			if ($node instanceof Node\Expr\ClosureUse) {
 				return [$node->var];
 			}
+			if ($node instanceof Node\Expr\FuncCall
```

Comment:
OK, so check that please. We have to make sure that we're not comparing a different object to string (there can also be `Expr`).